docs/UserGuide.md

```diff
@@ -106,23 +106,12 @@ Here's a quick look at the different parts of our GUI and some tips on how to us
 
 Adds a **person/ staff / supplier / maintainer** contact to the Pooch Planner.
 
-#### Adds a person
-Format: `/pooch-add ; name : [name] ; phone : [phone] ; address : [address] ; email : [email] ;`
-
-#### Adds a staff
-Format: `/pooch-staff ; name : [name] ; phone : [phone] ; address : [address] ; email : [email] ; salary : [salary/hr]  ; employment : [part/full] ;`
-
-#### Adds a supplier
-Format: `/pooch-supplier ; name : [name] ; phone : [phone] ; address : [address] ; email : [email] ; product : [product] ; price : [price/(quantity)] ;`
-
-#### Adds a maintainer
-Format: `/pooch-maintainer ; name : [name] ; phone : [phone] ; address : [address] ; email : [email] ; skill : [skill] ; commission : [commission/hr] ;`
-
-Examples:
-* `/pooch-add ; name : Janna  ; phone : 98765435 ; address : Poochie Street 24 ; email : ihelppooches@gmail.com`
-* `/pooch-staff ; name : Poochie ; phone : 98765435 ; address : Poochie Street 21 ; email : ilovecatstoo@gmail.com ; salary : $50/hr ; employment : part-time`
-* `/pooch-supplier ; name : PetCo ; phone : 98673098 ; address : Meow Street 24 ; email : ilovewombatstoo@gmail.com ; product : kibble ; price : $98/bag`
-* `/pooch-maintainer ; name : Tom Tan  ; phone : 98765435 ; address : Poochie Street 24 ; email : ihelppooches@gmail.com ; skill : trainer ; commission : $60/hr`
+| Adds a ...        | Format & Examples                                                                                                                                                                                                                                                                                                                  |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Person**        | Format : `/pooch-add ; name : [name] ; phone : [phone] ; address : [address] ; email : [email]` <br> Example: `/pooch-add ; name : Janna  ; phone : 98765435 ; address : Poochie Street 24 ; email : ihelppooches@gmail.com`                                                                                                       |
```

Comment:
Maybe can keep the explanation for the examples. Shouldn't the commands be /add-person, /add-staff etc.